test/functional/toolchains/cmake/test_cmake_multi.py

```diff
@@ -0,0 +1,113 @@
+        conanfile = textwrap.dedent("""
+            from conan import ConanFile
+            from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+            import os
+
+
+            class multiRecipe(ConanFile):
+                settings = "os", "compiler", "build_type", "arch"
+
+                exports_sources = "cmake_one/CMakeLists.txt", "cmake_two/CMakeLists.txt", "src_one/*", "src_two/*"
+
+                def layout(self):
+                    cmake_layout(self)
+
+                def generate(self):
+                    deps = CMakeDeps(self)
+                    deps.generate()
+                    tc = CMakeToolchain(self)
+                    tc.generate()
+
+                def build_one(self):
+                    cmake = CMake(self)
+                    cmake.configure(build_script_folder="cmake_one", build_subfolder="one")
+                    cmake.build(build_subfolder="one")
+                    # cmake.install(build_subfolder="one")
+
+                def build_two(self):
+                    cmake = CMake(self)
+                    # CMAKE_PREFIX_PATH
+                    cmake.configure(build_script_folder="cmake_two", build_subfolder="two")
+                    cmake.build(build_subfolder="two")
+
+                def build(self):
+                    self.build_one()
+                    self.build_two()
+
+                def package(self):
+                    cmake = CMake(self)
+                    cmake.install(build_subfolder="two")
+
+                def package_info(self):
+                    self.cpp_info.libs = ["hello_two"]
+                    self.cpp_info.includedirs = ['two/include']
+                    self.cpp_info.libdirs = ['two/lib']
+            """)
+
+        hello_cpp = textwrap.dedent("""
+            #include <iostream>
+            #include "hello_{name}.h"
+
+            void hello_{name}() {{
+                std::cout << "Hello, World {name}!" << std::endl;
+            }}
+            """)
+
+        hello_h = textwrap.dedent("""
+            #ifndef HELLO_{name}_H
+            #define HELLO_{name}_H
+
+            void hello_{name}();
+
+            #endif
+            """)
+
+        cmakelist = textwrap.dedent("""
+            cmake_minimum_required(VERSION 3.15)
+            project(hello_{name} LANGUAGES CXX)
+
+            # find_package(hello_one REQUIRED CONFIG)
+
+            add_library(hello_{name} ../src_{name}/hello_{name}.cpp)
+            target_include_directories(hello_{name} PUBLIC ../src_{name})
+
+            # target_link_libraries(hello_two PRIVATE hello_one)
+
+            set_target_properties(hello_{name} PROPERTIES PUBLIC_HEADER "../src_{name}/hello_{name}.h")
+            install(TARGETS hello_{name})
+            # install(TARGETS hello_one EXPORT one_target)
+            # install(EXPORT one_target DESTINATION lib FILE hello_one.cmake)
+            """)
+
+        client = TestClient(path_with_spaces=False)
```

Comment:
```suggestion
        client = TestClient()
```
Why? Everything should work with paths with spaces too

Reply:
copy paste from another test